src/address/channel.rs

```diff
@@ -555,6 +562,22 @@ impl<A: Actor> WeakAddressSender<A> {
     }
 }
 
+impl<A, M> WeakSender<M> for WeakAddressSender<A>
+where
+    A: Handler<M>,
+    A::Context: ToEnvelope<A, M>,
+    M::Result: Send,
+    M: Message + Send + 'static,
+{
+    fn upgrade(&self) -> Option<Box<dyn Sender<M> + Sync>> {
```

Comment:
I believe we should have a doc note about when this `Option` would be `None`, either here or at the trait's definition.

Reply:
sure, but probably at the definition of the trait and not here right?

Reply:
Both will be generated to the doc, but I think the trait it might be more noticeable. So if the `None` has a generic cause it would be better to have it there, if it's specific to the type's implementation we can have it here.

Reply:
I added it on the trait, but it's `pub(crate)`, so it won't show up in any docs anyway.
Plus: the semantics are basically the same as for `Weak<T>::upgrade()`.
Alright by you @Jonathas-Conceicao ?

Reply:
I added it on the trait, but it's pub(crate), so it won't show up in any docs anyway.
Plus: the semantics are basically the same as for Weak<T>::upgrade().
Alright by you @Jonathas-Conceicao ?